Detectors/TPC/monitor/macro/RunCompareMode3.C

```diff
@@ -32,7 +32,7 @@ void RunCompareMode3(TString fileInfo)
 
   auto arrData = fileInfo.Tokenize("; ");
   uint64_t checkedAdcValues = 0;
-  for (auto o : *arrData) {
+  for (const auto &o : *arrData) {
     const TString& data = static_cast<TObjString*>(o)->String();
```

Comment:
I'm not sure if in this case the &o is correct or necessary since (*arr) data will return TObject*, so &o would be a reference to a pointer. I think I didn't comment on that one. But perhaps it is fine.
The const auto& is important in case the objects are on the stack in order to not make a copy, but use them in place. For arrays of pointers you anyhow only get the pointer ...